Added table to show residents their submitted vacation requests

Sam41704 opened the pull request:
File edited: page.tsx
Why: defined userId

File edited: RequestOffPage.tsx
Why: input userId into = api/vacations/filter?residentId=${encodeURIComponent(userId) Added functionality to display table of users time off requests and status at bottom of page

Sam41704 committed:
Added table to show residents their submitted vacation requests

File edited: page.tsx
Why: defined userId

File edited: RequestOffPage.tsx
Why: input userId into = api/vacations/filter?residentId=${encodeURIComponent(userId)
Added functionality to display table of users time off requests and status at bottom of page

aitanoffir reviewed (approved):
looks good

aitanoffir reviewed (changes requested):
On lines 91 and 104 you are using "any" which is causing a typescript violation and not allowing the front end to build. Please fix

Sam41704 committed:
Merge branch 'main' into PSY-50-Show-time-off-requests-and-status-on-resident-side

Sam41704 committed:
Errors resolved

- Changed approach to avoid use of "any".
- Also was missing userId definition in dashboard\request-off\page.tsx so I added that.
- Ran frontend build and verified errors were resolved.

aitanoffir reviewed (approved):
TypeScript errors are resolved and the submitted requests look great and are functional
